### DISCUSSION TITLE
How to set up the operation materials for implementing alternate part?

### DISCUSSION THREAD

**Discussion by sgfisme** [frePPLe/frepple#438] [Q&A] [answered]
I added two start components into a operation, and give them different priority for applying alternate components. But the planner consume them all to product end item. 

If I hoped consume the high priority component first, not consume them all, is the only way that use the different priority operation to implement it? What is the meaning of the priority on the operation material?

**Comment by jdetaeye** [comment]
Check out https://frepple.org/docs/current/examples/buffer/alternate-materials.php

**Reply by sgfisme** [reply]
Thank you @jdetaeye .
I configured alternate materials based on your suggestion. Just wondering to know the planner will consume 2 parts together or choose any part that has WIP's if I set the same priority on the same name group?

**Comment by jdetaeye** [comment, marked as answer]
Frepple only supports item-level alterntes eg "consume A or B or C or D".

Group alternates are not supported:  we can't model "consume A+B or C+D".
